poppy/fresnel.py

```diff
@@ -256,6 +276,11 @@ def __init__(self,
         if self.oversample < 2:
             _log.warn("Oversampling > 2x suggested for reliable results.")
 
+        self._y, self._x = np.indices(self.shape, dtype=float)
+        self._y -= (self.wavefront.shape[0]) / 2.0
+        self._x -= (self.wavefront.shape[1]) / 2.0
+        """saves x and y indices for future use"""
```

Comment:
This is interesting. I'm guessing you decided to save these because benchmarking showed a significant enough cost in all the calls to generate the coordinates?  I'm a little surprised if there's substantial savings from doing this since you still have to do the multiplications by the pixel scales in the coordinates functions. But I have to admit I didn't actually bench mark it myself